dev/src/watch.ts

```diff
@@ -53,7 +54,7 @@ export const WATCH_IDLE_TIMEOUT_MS = 120 * 1000;
 /*!
  * Sentinel value for a document remove.
  */
-const REMOVED = {} as DocumentSnapshotBuilder<unknown>;
+const REMOVED = {} as DocumentSnapshotBuilder<any, any>;
```

Comment:
Yuck. I guess this is fine since it's an internal implementation detail. A `Symbol` would be more appropriate IMO (but that 's a change for another PR).